src/MicrodataDOMElement.php

```diff
@@ -176,15 +179,15 @@ public function tokenizeAttribute(string $attributeName)
     /**
      * Splits given attribute value in space characters to array
      *
+     * @param string $attribute
+     *
+     * @return array|bool
```

Comment:
Type hints don't support false, but we can at least specify it in DocBlocks.

```suggestion
     * @return array|false
```